content/docs/1.13/operator.md

```diff
@@ -421,6 +421,8 @@ spec:
 
 A complete sample deployment is available at [`deploy/examples/business-application-injected-sidecar.yaml`](https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/examples/business-application-injected-sidecar.yaml).
 
+When the sidecar is injected, the Jaeger Agent can then be accessed on `localhost`.
```

Comment:
Maybe change "accessed on" to "at its default location on"? As by default the jaeger clients will assume the agent is running on localhost.
Same change to be applied in 1.14/next-release.